label_studio_converter/imports/yolo.py

```diff
@@ -117,12 +142,14 @@ def convert_yolo_to_ls(input_dir, out_file,
         with open(out_file, 'w') as out:
             json.dump(tasks, out)
 
-        print('\n'
-              f'  1. Create a new project in Label Studio\n'
-              f'  2. Use Labeling Config from "{label_config_file}"\n'
-              f'  3. Setup serving for images [e.g. you can use Local Storage (or others):\n'
-              f'     https://labelstud.io/guide/storage.html#Local-storage]\n'
-              f'  4. Import "{out_file}" to the project\n')
+        print(
+            '\n'
+            f'  1. Create a new project in Label Studio\n'
+            f'  2. Use Labeling Config from "{label_config_file}"\n'
+            f'  3. Setup serving for images [e.g. you can use Local Storage (or others):\n'
+            f'     https://labelstud.io/guide/storage.html#Local-storage]\n'
+            f'  4. Import "{out_file}" to the project\n'
+        )
```

Comment:
This should be logging, no?

Reply:
yes, we can do it as logging, but we should be sure that users will see it 100%